src/gui/src/dbDescriptors.cpp

```diff
@@ -403,6 +404,127 @@ bool DbInstDescriptor::getAllObjects(SelectionSet& objects) const
   return true;
 }
 
+std::string DbInstDescriptor::getInstanceTypeText(Type type) const
+{
+  switch (type) {
+  case BLOCK:
+    return "Macro";
+  case PAD:
+    return "Pad";
+  case ENDCAP:
+    return "Endcap";
+  case FILL:
+    return "Fill";
+  case TAPCELL:
+    return "Tapcell";
+  case BUMP:
+    return "Bump";
+  case COVER:
+    return "Cover";
+  case ANTENNA:
+    return "Antenna";
+  case TIE:
+    return "Tie";
+  case LEF_OTHER:
+    return "Other";
+  case STD_CELL:
+    return "Standard cell";
+  case STD_BUFINV:
+    return "Standard cell buffer/inverter";
+  case STD_BUFINV_CLK_TREE:
+    return "Standard cell clock buffer/inverter";
```

Comment:
The prefix "Standard cell"  tends to take up a lot of space.  I think it isn't necessary

Reply:
Removed